scripts/build-last-tracing-runtime.sh

```diff
@@ -11,6 +11,5 @@ git clone --depth 1 -b master-without-wasm https://github.com/PureStake/moonbeam
 
 cd build/moonbeam-runtime-overrides
 ./scripts/import-tracing-runtime.sh local ${1:-"$LOCAL_GIT_BRANCH"}
-cd tracing/local && cargo update -p evm && cd ../..
```

Comment:
Why this?

Reply:
It seemed to be necessary at some point but no longer is, and instead doesn't seem to like the git dependency.